packages/feedback/src/core/integration.ts

```diff
@@ -279,5 +282,3 @@ export const _feedbackIntegration = (({
     },
   };
 }) satisfies IntegrationFn;
-
-export const feedbackIntegration = defineIntegration(_feedbackIntegration);
```

Comment:
Nice not to need this anymore!

Reply:
for reference, the reason we generally have this is because for most integrations, we do not want to expose to users what shape they have, as then we cannot refactor internals anymore - we only expose it if we have to (which we do for replay & feedback) :)